dali/test/python/test_operator_normalize.py

```diff
@@ -6,17 +6,24 @@
 import numpy as np
 from test_utils import dali_type
 
-def normalize(x, axes = None, mean = None, stddev = None):
+def normalize(x, axes = None, mean = None, stddev = None, ddof = 0, eps = 0):
     if type(axes) is list:
         axes = tuple(axes)
-    if mean is None and stddev is None:
-        mean = x.mean(axis = axes, keepdims = True)
-        stddev = np.std(x, axis = axes, keepdims=True)
-    elif mean is None:
+
+    if mean is None:
         mean = x.mean(axis = axes, keepdims = True)
-    elif stddev is None:
-        var = ((x - mean).astype(np.float)**2).mean(axis = axes, keepdims = True)
-        stddev = np.sqrt(var)
+
+    if stddev is None:
```

Comment:
Maybe you can just use `numpy.std` at least for some case?

Reply:
It's going to be a bit artificial, but I can give it a shot.

Reply:
I'm asking to validate if or definition of ddof matches the one from numpy. But it is just a suggestion.

Reply:
Done.